SSNFSd/ssnfsworker.cpp

```diff
@@ -171,6 +173,22 @@ QString SSNFSWorker::getPerms(QString path, qint32 uid) {
     }
 }
 
+ QByteArray SSNFSWorker::isValidPath(QByteArray path){
```

Comment:
Why return a QByteArray? This function just performs a boolean (true/false only) check.